### PR TITLE
Feature/ruud #11 [WIP]

### LINKED ISSUE #11: Log Receiver should only support POST operations
Given the server is running
When I do a HTTP GET to '/' on the LogReceiver port
Then the response should have statuscode 405

### PR DESCRIPTION
**Pull request by ruud**
closes #11

### CODE REVIEW

**Comment by jordi133** [comment]
I put a comment on the user story "Acceptance test should be implemented in LogSubmissionTest.scala". Did you forget this?

**Comment by ruud** [comment]
Just changed test to LogSubmissionTest.scala

**Comment by jordi133** [comment]
My comment on the user story was:

> **Acceptance test** should be implemented in **LogSubmissionTest.scala**. Keep in mind that **unit testing** for the HTTP routes should be done in **LogReceiverSpec** similar to AdministratorSpec

Both unit tests **and** integration tests are usefull to add. In a unit test you can look at the specifically the route, just as you have done using the ScalatestRouteTest. In the integration test, the idea is to just start the application and approach the application just as it would be approached in production, through a http-call to the server.

**Comment by ruud** [comment]
Ok, that is clear. Will add an integration test.

File: src/test/scala/performanceanalysis/logreceiver/LogSubmissionTest.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package performanceanalysis.logreceiver
```
Comment (geerdink) [inline src/test/scala/performanceanalysis/logreceiver/LogSubmissionTest.scala:1]: Should be in the 'it' folder.

File: src/test/scala/performanceanalysis/logreceiver/LogSubmissionTest.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import performanceanalysis.base.SpecBase
+
+/**
+  * Created by Ruud on 12-04-16.
```
Comment (TimSoethout) [inline src/test/scala/performanceanalysis/logreceiver/LogSubmissionTest.scala:7]: I guess we can leave out this comment, or replace it with documentation

**Pull request closed without merging**